sfaira/data/base.py

```diff
@@ -481,17 +481,14 @@ def _collapse_gene_versions(self, remove_gene_version):
                 # last element of each block as block boundaries:
                 # n_genes - 1 - idx_map_sorted_rev.index(x)
                 # Note that the blocks are named as positive integers starting at 1, without gaps.
-                counts = np.concatenate([
-                    np.sum(x, axis=1, keepdims=True)
-                    for x in np.split(
-                        self.adata[:, idx_map_sorted_fwd].X,  # forward ordered data
-                        indices_or_sections=[
-                            n_genes - 1 - idx_map_sorted_rev.index(x)  # last occurrence of element in forward order
-                            for x in np.arange(0, len(new_index_collapsed) - 1)  # -1: do not need end of last partition
-                        ],
-                        axis=1
-                    )
-                ][::-1], axis=1)
+                counts = np.concatenate([np.sum(x, axis=1, keepdims=True)
```

Comment:
Think the formatting "broke" when I used Pycharms autoformatting. 
But the way it is now is also fine.